core/commands/partials/vagrant_configuration_generator.rb

```diff
@@ -298,10 +294,34 @@ def get_role_description(name, product, box)
                        {}
                      end
     @ui.info("Recipe #{recipe_name}")
-    make_role_json(name, product_config, recipe_name, box)
+    [recipe_name, product_config]
```

Comment:
I suggest to return the hash here instead of the array. This will make the result to be easily managed.

```ruby
{
  name: recipie_name,
  config: product_config
}
```